.github/workflows/test.yml

```diff
@@ -108,13 +197,29 @@ jobs:
         run: 'gz sim --versions'
 
   test_gazebo_install_ubuntu_nightly:
-    name: 'Check installation of Gazebo nightly binary on Ubuntu'
+    name: 'Install Gazebo nightly on Ubuntu GitHub runner'
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4.0.3
+        with:
+          node-version: '20.x'
+      - name: 'Check Gazebo installation on Ubuntu runner'
+        uses: ./
+        with:
+          required-gazebo-distributions: 'ionic'
+          use-gazebo-nightly: 'true'
+      - name: 'Test Gazebo installation'
+        run: 'gz sim --versions'
+
+  test_gazebo_install_ubuntu_nightly_docker:
```

Comment:
We can leave the version of running the pre-release and nightly tests on docker containers and remove the github runners. Do not need to test it twice in different environments, looks to me like too much testing that does not differ too much.